packages/react-charting/src/components/CommonComponents/ErrorBoundary.types.tsx

```diff
@@ -0,0 +1,105 @@
+import { IStyle, IStyleFunctionOrObject } from '@fluentui/react';
+import { ITheme } from '@fluentui/react/lib/Styling';
+
+export interface IErrorBoundaryProps {
+  /**
+   * If true, the error boundary will show the empty state.
+   */
+  hasEmptyState: boolean;
+
+  /**
+   * Custom error state message.
+   */
+  customErrorMsg?: string;
```

Comment:
call it just errorMessage

Reply:
And how will you handle error message and empty data message individually.

Reply:
Added provision separately for custom empty message and added corresponding tests.